README.md

```diff
@@ -4,12 +4,20 @@ This is a base snap for snapd that is based on Ubuntu 18.04
 
 # Building locally
 
-To build this snap locally you need snapcraft. The project must be built as real root.
+To build this snap locally you need snapcraft. The project must be built as real root. 
```

Comment:
Is the requirement to build as root user still true?

Reply:
unfortunately yes, as one needs to create and package device nodes and binaries with setuid capabilities set.